src/frontend-samples/issues-sample/IssuesApi.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import { AuthorizedFrontendRequestContext,IModelApp } from "@bentley/imodeljs-frontend";
+import { MarkerData, MarkerPinDecorator } from "../marker-pin-sample/MarkerPinDecorator";
+import { Point3d } from "@bentley/geometry-core";
+import { IssueGet, IssuesClient } from "./IssuesClient";
+import { AuthorizationClient } from "@itwinjs-sandbox";
+
+export default class IssuesApi {
+  private static _issueClient: IssuesClient<unknown>;
```

Comment:
What is `<unknown>`  Supposed to be some kind of SecurityDataType?

Reply:
This has been fixed. The ApiConfig had to be defined and passed into the client.

```
const config: ApiConfig<string> = {
    securityWorker: (accessToken: string | null) => accessToken ? { headers: { ["Authorization"]: token } } : undefined,
 };
```